src/main/java/com/alkemy/ong/infrastructure/rest/mapper/SlideMapper.java

```diff
@@ -0,0 +1,26 @@
+package com.alkemy.ong.infrastructure.rest.mapper;
+
+import com.alkemy.ong.domain.Slide;
+import com.alkemy.ong.infrastructure.rest.response.ListSlideResponse;
+import com.alkemy.ong.infrastructure.rest.response.SlideResponse;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SlideMapper {
```

Comment:
Este mapper se debería llamar ListSlideMapper según el estándar que definieron en los endpoint que existen en main. 

Además, ListSlideMapper(from SlideResponse to ListSlideResponse) debería llamar a SlideMapper (from Slide to SlideRespose).